### ISSUE TITLE
Cannot compile ROLE_PERIPHERAL disabled program

### ISSUE THREAD

**Issue by homy-newfs8** [h2zero/NimBLE-Arduino#742]
Hi,

Compile error occurred on ROLE_PERIPHERAL disabled program. Simplified code:
```c++
#define CONFIG_BT_NIMBLE_ROLE_PERIPHERAL_DISABLED
#include <NimBLEDevice.h>

void
setup() {}
void
loop() {}
```
Compiler message:
> NimBLEDevice.h:138:52: error: 'NimBLEConnInfo' does not name a type; did you mean 'BLEConnInfo'?

**Comment by h2zero** [comment]
Thank you for reporting this, I will fix it shortly.

**Comment by homy-newfs8** [comment]
Fix confirmed on 68b9e266c843b58df00b8787c1c384584b9180f9.
I appreciate your efforts!
